packages/core/src/components/button/test/button.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { register, render } from '@stencil/core/testing';
+import { Button } from '../button';
+
+describe('button', () => {
+  let plt: any;
+  beforeEach(() => {
+    plt = register([{
```

Comment:
I am thinking of making the current platform essentially a static local in `@stencil/core/testing` so it doesn't need to be passed around like this. Thoughts?

My thought is if I did that, combined with handling Josh's upcoming meta-data mods, this code becomes something like:

```ts
beforeEach(() => {
  register([Button]);
});
```

Then subsequent calls to `render()` would just use whatever platform was last registered.